module/userExtension/helper.js

```diff
@@ -525,10 +525,7 @@ module.exports = class UserExtensionHelper {
             }, { "roles.acl" : 1 }).lean();
 
             if (!userExtensionDoument) {
-                throw { 
-                    status: httpStatusCode.badrequest.status, 
-                    message: messageConstants.apiResponses.USER_EXTENSION_NOT_FOUND 
-                };
+                return resolve({});
```

Comment:
@aman-tunerlabs - Not a good idea to return blank objects, use flags or status for the same.